apps/predictions/test/repo_test.exs

```diff
@@ -274,4 +274,40 @@ defmodule Predictions.RepoTest do
 
     assert log =~ "Discarding prediction"
   end
+
+  test "drops subway prediction if it is in the past" do
```

Comment:
This test is great. Just because it is important functionality, could you please also add a test showing that past predictions are _not_ dropped for the commuter rail?